src/odbc/unittests/utf8_2.c

```diff
@@ -71,9 +80,26 @@ TEST_MAIN()
 		odbc_command(tmp);
 	}
 
-	/* test conversions in libTDS */
+	/* test conversions in libTDS
+	 *
+	 * Because column type is VARCHAR with collation defined, SQL Server returns
+	 * the data as CP1255 (hebrew) or GB18030 (CN), and FreeTDS's built-in iconv
+	 * doesn't support those. So this test fails in Windows where libiconv is not
+	 * really supported, although it works in Linux.
+	 *
+	 * Using NVARCHAR makes this test work as it causes the server to return the
+	 * data in wide characters. This could also be done in the CREATE TABLE,
+	 * using NVARCHAR instead of VARCHAR.
+	 *
+	 * Changing the collation type to a UTF-8 collation would fix the SELECT,
+	 * however the INSERT statements would need to be modified.
+	*/
+#if HAVE_ICONV
 	odbc_command("SELECT hebrew, cn FROM #tmp ORDER BY i");
-
+#else
+	fprintf(stderr, "Bypassing Hebrew MBCS test since iconv not installed (Forcing server to send UTF-8).\n");
```

Comment:
The server will send UTF-16, not UTF-8, it's the client that converts to UTF-8, as built-in iconv supports both.